src/lib/y2storage/boot_requirements_strategies/analyzer.rb

```diff
@@ -223,6 +223,14 @@ def efi_in_md_raid1?
         return raid.md_level.is?(:raid1)
       end
 
+      # Method to return all prep partitions - newly created and also reused.
+      # It is useful to do checks on top of that partitions
+      def prep_partitions
```

Comment:
`BootRequirements::Analyzer` is designed in a way that most of their methods analyze the full set of devices (i.e. the planned devices + the devices on the devicegraph). Methods that only search in part of the whole set (this one search the devicegraph but ignores planned_devices) should have a name that makes that clear.

In other words, I would expect a method `Analyzer#prep_partitions` to return a superset of `Analyzer#planned_prep_partitions" and the PReP partitions in the devicegraph.

For the partitioner that's irrelevant because planned_partitions is empty in that scope. But if the proposal or AutoYaST initialize the analyzer object as `Analyzer.new(devicegraph, some_planned_devices)` they expect methods to check both unless the name of the method says otherwise.

Reply:
so should I rename it, move it or add also planned ones? what do you suggest?

Reply:
what about rename it to something like `graph_prep_partitions` ? and if you want both you do `graph_prep_partitions + planned_prep_partitions`

Reply:
Fine for me